index.html

```diff
@@ -9,37 +9,88 @@
       href="https://fonts.googleapis.com/css2?family=EB+Garamond:ital,wght@0,400..800;1,400..800&family=Sora:wght@100..800&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.0/css/all.min.css"
+    />
     <title>Weather-app</title>
     <link rel="stylesheet" href="./styles.css" />
   </head>
-  <body>
+  <body onload="getUserLocation()">
     <div class="container">
-      <div class="today">
-        <form action="" id="search-form">
-          <input type="text" id="search-input" placeholder="Ask Me AnyWhere"/>
-        </form>
-        <p class="today-city">Ho Chi Minh</p>
-        <button id="change-temperature">0C / 0F</button>
-        <div class="today-temperature-wrapper">
-          <span class="today-temperature">26</span>
+      <div class="container-inner">
+        <div class="today">
+          <form action="" id="search-form">
+            <input
+              type="text"
+              id="search-input"
+              placeholder=""
+            />
+            <select id="change-temperature" onclick="convert()">
```

Comment:
The hint:  to make  convert temperature feature start working is to use `onchange` event instead of `onclick`